openedx/adg/lms/webinars/tests/test_admin.py

```diff
@@ -268,3 +268,14 @@ def test_registration_webinar_admin_delete_permission():
     Test that admins do not have permission to delete webinar registrations
     """
     assert not WebinarRegistrationAdmin.has_delete_permission('self', Mock())
+
+
+@pytest.mark.django_db
+def test_get_deleted_objects_webinar_admin(webinar_admin_instance, request, webinar):
+    """
+    Test that get_deleted_objects for webinar admin returns an empty set of permissions
+    """
+    request.user = UserFactory()
+    deleted_objects, model_count, perms_needed, protected = \
+        webinar_admin_instance.get_deleted_objects([webinar], request)
```

Comment:
We can format this a little better. 
```
deleted_objects, model_count, perms_needed, protected = webinar_admin_instance.get_deleted_objects(
        [webinar], request
)
```

Reply:
I had the following lines in mind from [pep8](https://www.python.org/dev/peps/pep-0008/#multiline-if-statements).

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

> Backslashes may still be appropriate at times. For example, long, multiple with-statements cannot use implicit continuation, so backslashes are acceptable

Reply:
```
    deleted_objects, model_count, perms_needed, protected = webinar_admin_instance.get_deleted_objects([webinar],
                                                                                                       request)
```